### PR TITLE
Adding Polls november 15 2021

### PR DESCRIPTION
**Pull request by prose11**
Note @mkrorbrkr there are 3 Offboarding Proposals to be added here so if you don't see them it's still WIP

### CODE REVIEW

**Comment by prose11** [comment]
Should be good to go now @mkrorbrkr 5 mins ahead of schedule ;)

File: governance/polls/MIP9 Community Greenlight Poll - Monetalis - November 15, 2021.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Community Greenlight Poll - MONETALIS (Montalis Wholesale SME Green Growth Lending) - November 15, 2021
```
Comment (mkrorbrkr) [inline governance/polls/MIP9 Community Greenlight Poll - Monetalis - November 15, 2021.md:2]: Monetalis spelled wrong in the brackets

File: governance/polls/MIP9 Community Greenlight Poll - Monetalis - November 15, 2021.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+summary: Signal your support or opposition to prioritising onboarding MONETALIS (Montalis Wholesale SME Green Growth Lending).
```
Comment (mkrorbrkr) [inline governance/polls/MIP9 Community Greenlight Poll - Monetalis - November 15, 2021.md:3]: Monetalis spelled wrong in the brackets

File: governance/polls/MIP9 Community Greenlight Poll - Monetalis - November 15, 2021.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+   - Collateral
+   - Greenlight
+options:
+   0: Abstain
+   1: Yes (Greenlight)
+   2: No (Defer)
+start_date: 2021-11-15T16:00:00
+end_date: 2021-11-29T16:00:00
+---
+# Poll: Community Greenlight Poll - MONETALIS (Montalis Wholesale SME Green Growth Lending) - November 15, 2021
```
Comment (mkrorbrkr) [inline governance/polls/MIP9 Community Greenlight Poll - Monetalis - November 15, 2021.md:16]: Monetalis spelled wrong in the brackets

File: governance/polls/MIP9 Community Greenlight Poll - Monetalis - November 15, 2021.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+The Governance Facilitators have placed a governance poll into the [voting system](https://vote.makerdao.com/polling) as part of their responsibilities as defined in [MIP9: Community Greenlight](https://mips.makerdao.com/mips/details/MIP9). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, November 15 at 16:00 UTC.
+
+**This is a binary vote.** 
+- **You may vote for a single option.** 
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+This poll asks whether the asset MONETALIS (Montalis Wholesale SME Green Growth Lending) should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state).
```
Comment (mkrorbrkr) [inline governance/polls/MIP9 Community Greenlight Poll - Monetalis - November 15, 2021.md:27]: Monetalis spelled wrong in the brackets

File: governance/polls/MIP9 Community Greenlight Poll - Monetalis - November 15, 2021.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+## Review
+
+This poll asks whether the asset MONETALIS (Montalis Wholesale SME Green Growth Lending) should be prioritized for inclusion in the Maker Protocol by the [Core Units](https://mips.makerdao.com/mips/details/MIP38#mip38c2-core-unit-state). 
+
+Please review the collateral onboarding application [thread](https://forum.makerdao.com/t/mip6-collateral-onboarding-application-monetalis-wholesale-green-economy-senior-secure-sme-funding/11223) to inform your position before voting.
+
+## Outcomes
+
+**Regardless of the winning option the following actions will be taken at the conclusion of this poll:**
+* The 'greenlight score' will be calculated for MONETALIS (Montalis Wholesale SME Green Growth Lending) using the formula `Score = 'Yes' Votes - 'No' Votes`
```
Comment (mkrorbrkr) [inline governance/polls/MIP9 Community Greenlight Poll - Monetalis - November 15, 2021.md:34]: Monetalis spelled wrong in the brackets

File: governance/polls/MIP9 Community Greenlight Poll - Monetalis - November 15, 2021.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+* The 'greenlight score' for MONETALIS (Montalis Wholesale SME Green Growth Lending) will be added to the [Collateral Prioritisation Sheet](https://docs.google.com/spreadsheets/d/1IX9e2fyfz7djtDMKn5gMyGsyFxHoY75GncMbAjnSXrM/edit#gid=0) by the Governance Facilitators.
```
Comment (mkrorbrkr) [inline governance/polls/MIP9 Community Greenlight Poll - Monetalis - November 15, 2021.md:35]: Monetalis spelled wrong in the brackets

File: governance/polls/Offboard AAVE-A - November 15, 2021.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
+---
+title: Offboard the AAVE-A Vault Type - November 15, 2021
+summary: Signal your support or opposition to offboard the AAVE-A vault type.
+discussion_link: https://forum.makerdao.com/t/signal-request-offboarding-matic-comp-aave-and-bal/11184
+vote_type: Plurality Voting
+categories:
+   - Collateral
```
Comment (mkrorbrkr) [inline governance/polls/Offboard AAVE-A - November 15, 2021.md:7]: Apparently there's an "Offboard" category - not essential but I guess we could include it?

https://github.com/makerdao/community/blob/master/governance/polls/meta/categories.json

File: governance/polls/Offboard BAL-A - November 15, 2021.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
+---
+title: Offboard the BAL-A Vault Type - November 15, 2021
+summary: Signal your support or opposition to offboard the BAL-A vault type.
+discussion_link: https://forum.makerdao.com/t/signal-request-offboarding-matic-comp-aave-and-bal/11184
+vote_type: Plurality Voting
+categories:
+   - Collateral
```
Comment (mkrorbrkr) [inline governance/polls/Offboard BAL-A - November 15, 2021.md:7]: Apparently there's an "Offboard" category - not essential but I guess we could include it?

https://github.com/makerdao/community/blob/master/governance/polls/meta/categories.json

File: governance/polls/Offboard COMP-A - November 15, 2021.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
+---
+title: Offboard the COMP-A Vault Type - November 15, 2021
+summary: Signal your support or opposition to offboard the COMP-A vault type.
+discussion_link: https://forum.makerdao.com/t/signal-request-offboarding-matic-comp-aave-and-bal/11184
+vote_type: Plurality Voting
+categories:
+   - Collateral
```
Comment (mkrorbrkr) [inline governance/polls/Offboard COMP-A - November 15, 2021.md:7]: Apparently there's an "Offboard" category - not essential but I guess we could include it?

https://github.com/makerdao/community/blob/master/governance/polls/meta/categories.json

File: governance/polls/Offboard AAVE-A - November 15, 2021.md [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes: 
+* Lower the AAVE-A [Liquidation Penalty (chop)](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%.
+* Lower the AAVE-A [Debt Ceiling (line)](https://makerdao.world/en/learn/governance/param-debt-ceiling) to 0 DAI.
+* Increase the AAVE-A [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) over 60 days using dss-lerp such that all vaults are liquidated by the end of that period.
+
```
Comment (mkrorbrkr) [inline governance/polls/Offboard AAVE-A - November 15, 2021.md:30]: Is it worth linking to the thread from Risk with the recommended offboarding parameters? Again can take it or leave it

https://forum.makerdao.com/t/proposed-offboarding-collateral-parameters-2/11548

File: governance/polls/Offboard BAL-A - November 15, 2021.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes: 
+* Lower the BAL-A [Liquidation Penalty (chop)](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%.
+* Lower the BAL-A [Debt Ceiling (line)](https://makerdao.world/en/learn/governance/param-debt-ceiling) to 0 DAI.
+* Increase the BAL-A [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) over 60 days using dss-lerp such that all vaults are liquidated by the end of that period.
+
```
Comment (mkrorbrkr) [inline governance/polls/Offboard BAL-A - November 15, 2021.md:30]: Is it worth linking to the thread from Risk with the recommended offboarding parameters? Again can take it or leave it

https://forum.makerdao.com/t/proposed-offboarding-collateral-parameters-2/11548

File: governance/polls/Offboard COMP-A - November 15, 2021.md [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes: 
+* Lower the COMP-A [Liquidation Penalty (chop)](https://makerdao.world/en/learn/governance/param-liquidation-penalty/) to 0%.
+* Lower the COMP-A [Debt Ceiling (line)](https://makerdao.world/en/learn/governance/param-debt-ceiling) to 0 DAI.
+* Increase the COMP-A [Liquidation Ratio](https://makerdao.world/en/learn/governance/param-liquidation-ratio/) over 60 days using dss-lerp such that all vaults are liquidated by the end of that period.
+
```
Comment (mkrorbrkr) [inline governance/polls/Offboard COMP-A - November 15, 2021.md:30]: Is it worth linking to the thread from Risk with the recommended offboarding parameters? Again can take it or leave it

https://forum.makerdao.com/t/proposed-offboarding-collateral-parameters-2/11548

**Review by mkrorbrkr — CHANGES_REQUESTED** [review]
@prose11 only 1 major issue which is the spelling of monetalis, otherwise made a few small, non-essential suggestions, see what you think

File: governance/polls/MIP9 Community Greenlight Poll - Monetalis - November 15, 2021.md (hunk 1 above)
Comment (prose11) [reply governance/polls/MIP9 Community Greenlight Poll - Monetalis - November 15, 2021.md:2]: That would have been bad!

File: governance/polls/Offboard BAL-A - November 15, 2021.md (hunk 8 above)
Comment (prose11) [reply governance/polls/Offboard BAL-A - November 15, 2021.md:7]: Pretty sure it's not compatible with the hard coded categories, but will follow up on that

File: governance/polls/Offboard AAVE-A - November 15, 2021.md (hunk 10 above)
Comment (prose11) [reply governance/polls/Offboard AAVE-A - November 15, 2021.md:30]: I like it :)

**Pull request merged**